cmake/projects/NASM/hunter.cmake

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2017, Zhuhao Wang
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+if(CMAKE_HOST_WIN32)
+  # NASM only supports a subset of VS compilers on Windows. However, the only
```

Comment:
You can use [MINGW](https://cmake.org/cmake/help/latest/variable/MINGW.html). `if(MINGW)` should do the trick.